marconi-cardano-core/src/Marconi/Cardano/Core/Indexer/SyncHelper.hs

```diff
@@ -3,7 +3,7 @@
 {-# LANGUAGE QuasiQuotes #-}
 
 -- | Provide standard query and helper functions to track the last sync points of an indexer
-module Marconi.ChainIndex.Indexers.SyncHelper (
+module Marconi.Cardano.Core.Indexer.SyncHelper (
```

Comment:
I think this module should go in `marconi-cardano-indexers`.

Reply:
I moved this one back to `marconi-chain-index`, so that we can move it to `marconi-cardano-indexers` when we create that one.